docs/installation/install_docker.md

```diff
@@ -0,0 +1,35 @@
+# Installing via Docker
+
+## Get Docker
+
+Docker is a containerization platform that allows you to run applications in a sandboxed environment. You can install Docker on your local machine or on a server. For more information, see the [Docker documentation](https://docs.docker.com/).
+
+You can download docker from here: [Get Docker](https://docs.docker.com/get-docker/).
+
+## Install Lute
+
+### Get the code
+
+You can get the code from the [GitHub repository](https://github.com/jzohrab/lute). Either git clone it or download the latest release ZIP file and unpack it.
+
+### Build and run the stack
+
```

Comment:
Should this project have people build the containers, or should I pre-build the images on release and put them on Docker Hub (or whatever it is ...) ?

Reply:
Actually, if it's a pre-built image, we'd have to handle all of the .env files differently, because different users can have different .env files, and the file couldn't be pre-baked into the image.  For example, for my tests, I sometimes like to have one db named test_lute, and then test_blah for other stuff.  Perhaps the .env would need to be mounted ... or the env settings passed to the container on startup.  Not sure!  Thoughts?